protoc_plugin/lib/indenting_writer.dart

```diff
@@ -27,6 +27,9 @@ class IndentingWriter {
   int _previousOffset = 0;
   final String? _sourceFile;
 
+  // Named text sections to write at the end of the file.
+  Map<String, String> suffixes = {};
```

Comment:
Should this be private? (`_suffixes`)

Since we iterate this map based on ordering of keys, can we use another type here that gives us ordered iteration by default? I think `SplayTreeMap` does that.

Reply:
I switched this to `_suffixes` (private);

Changing to using a type w/ ordering built-in makes sense; I updated the type we create to `SplayTreeMap` and removed the explicit key sorting where we read from the map below.